Packs/HackerOne/Integrations/HackerOne/HackerOne.yml

```diff
@@ -128,15 +128,13 @@ script:
       required: false
       secret: false
     - default: false
-      defaultValue: '50'
       description: "The number of reports to retrieve per page. \n\nNote: Possible\
         \ values are between 1 and 100."
       isArray: false
       name: page_size
       required: false
       secret: false
     - default: false
-      defaultValue: '0'
```

Comment:
Why have we deleted this? Plus shouldn't it be `1`?

Reply:
Need to mention in the description that the default is 1 (you are enforcing it in the code)